apps/accounts/models.py

```diff
@@ -378,30 +319,9 @@ class Meta:
     def __str__(self):
         return self.code
 
-    def url(self):
-        hostname = getattr(settings, 'HOSTNAME_URL', 'http://localhost:8000')
-
-        if "http://" in hostname.lower():
-            pass
-        elif "https://" in hostname.lower():
-            pass
-        else:
-            logger.debug("HOSTNAME_URL [%s] "
-                         "does not contain http or https prefix. "
-                         "Issuer:%s" % (settings.HOSTNAME_URL, hostname))
-            # no http/https prefix in HOST_NAME_URL so we add it
-            hostname = "https://%s" % (hostname)
-        return "%s%s" % (hostname,
-                         reverse('accounts_create_account'))
-
     def save(self, commit=True, **kwargs):
         if commit:
-            if self.valid:
-                # send the invitation verification email.
-                send_invite_to_create_account(self)
-                logger.info("Invitation sent to {}".format(self.email))
-
-            super(Invitation, self).save(**kwargs)
+            super().save(**kwargs)
```

Comment:
can these models just go away completely?

Reply:
This sounds good to me. I'll try removing from the models.py, but without running related migrations.

Reply:
The models are now removed.